packages/cli/bin/cli.js

```diff
@@ -41,6 +41,14 @@ notifier.notify({
   message: pkg.name === '@hubspot/cms-cli' ? CLI_UPGRADE_MESSAGE : null,
 });
 
+const getTerminalWidth = () => {
+  const width = yargs.terminalWidth();
+
+  if (width >= 100) return width * 0.9;
```

Comment:
0.8 ended up seeming like a lot of whitespace, so I bumped it to .9

Also, the way the terminal's width is measured is in columns, that's why the width is so low